### ISSUE TITLE
[Feature Request] Implementation of common Office short-cuts, mainly "F4"

### ISSUE THREAD

**Issue by rustysys-dev** [ONLYOFFICE/DocumentServer#1943]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
No support for many common short cuts.

**What is the expected behavior?**
As in WPS Office or MS Office, I would like to see many of the same short-cuts for the editors.  Namely, the implementation of the "F4" ( repeat last action ) short-cut.  This is paramount for formatting as it drastically reduces the amount of time required.

**Which versions of DesktopEditors, and which OS are affected by this issue? Did this work in previous versions of DesktopEditors?**
I have installed only the newest version, but I would assume that if this version doesn't have that feature then neither do the others... correct me if I am wrong.

**Comment by Lev-Shereshevsky** [comment]
I really miss this feature! In some cases it can save many clicks and time.

**Comment by EmmanuelIT** [comment]
+1.

F4 shortcut is needed, mainly for inserting new rows

**Comment by juhnny** [comment]
Hope this feature to be added soon.

**Comment by s-rades** [comment]
+1
We will use F4 shortcut to repeat formatting a cell/row

**Comment by barbouff** [comment]
Hello, I'm also interested in this topic, and if it helps, I'd like to share my perspective as an average user (I'll try to be clear—if I'm not, blame Deepl :)

I used this feature in Excel on my Windows PC with the keyboard shortcut Ctrl+Y (F4 on Mac), particularly for inserting rows—all I had to do was insert one row and then press Ctrl+Y as many times as I wanted to insert additional rows. 

Today, with OnlyOffice on Linux, I’ve found a feature that’s almost the same; for example, if I insert a row and press Ctrl+Z, the backspace key works and the row is deleted, and if I then press Ctrl+Y, the row is restored. So the Ctrl+Y command already works since it knows what to do after a backspace; however, it doesn’t know how to initiate an action if it hasn’t been “fed” beforehand by the content of the previous command.

To summarize:
- How it works now: I perform an action (and it’s probably stored somewhere), because after undoing it (Ctrl+Z), I can redo it—but only once (Ctrl+Y).
- What would be great: I perform an action, and I can redo it as many times as I want (Ctrl+Y).
Ctrl+Z works well; it undoes as many steps as needed.

So, obviously, we need to..., all that’s left is to... :)

Good luck to the whole team!

Translated with [DeepL.com](https://www.deepl.com/?utm_campaign=product&utm_source=web_translator&utm_medium=web&utm_content=copy_free_translation) (free version)
